src/tiledb/cloud/groups.py

```diff
@@ -92,6 +94,45 @@ def info(uri: str) -> object:
     return groups_client.get_group(group_namespace=namespace, group_name=group_name)
 
 
+def update_info(
+    uri: str,
+    *,
+    description: Optional[str] = None,
+    name: Optional[str] = None,
+    logo: Optional[str] = None,
+    tags: Optional[List[str]] = None,
+) -> None:
+    """
+    Update Group Metadata
```

Comment:
nit: Maybe "Update Group Attributes", metadata has a very specific meaning in the groups/array context. The term generally refers to the on-disk metadata stored alongside arrays/group

Reply:
Updated in the latest commit from "metadata" to "attributes"